index.html

```diff
@@ -21,6 +21,11 @@
 						<a href="./index.html" class="logo">Logo</a>
 					</div>
 					<ul class="col-8 nav justify-content-end">
+						<li>
+							<button onclick="myFunction()">
```

Comment:
you can transfer this logic in Js
`const btn = document.querySelector("button")`
QuerySelector can get by anything, class, id, element, brackets whatever you pass. With class it will be 
const btn = document.querySelector(".someClass")

Additionally you can switch to querySelector when you get the elementById in the code above.